site/_docs/history.md

```diff
@@ -51,21 +50,426 @@ using JDK/OpenJDK versions 8 to 19;
 Guava versions 16.0.1 to 31.1-jre;
 other software versions as specified in gradle.properties.
 
+#### New features
+{: #new-features-1-36-0}
+
+#### Dependency version upgrade
+{: #dependency-1-36-0}
+
+#### Bug-fixes, API changes and minor enhancements
+{: #fixes-1-36-0}
+
+#### Build and test suite
+{: #build-1-36-0}
+
+#### Web site and documentation
+{: #site-1-36-0}
+-->
+
+## <a href="https://github.com/apache/calcite/releases/tag/calcite-1.35.0">1.35.0</a> / 2023-07-13
+{: #v1-35-0}
+
+This release comes 4 months after [1.34.0](#v1-34-0),
+contains contributions from 36 contributors, and resolves 139 issues.
+
+Among others new features, we have added extensive function support. For example, we added roughly 14 new functions in BigQuery library for handling array, dates, math, and string.
+Also, we added 27 new functions in Spark library for handling array, map and about 11 new functions for other database products.
+
+Among other improvements and fixes, it is worth highlighting the following improvements:
+
+* Some improvements in calcite core.
+  * [<a href="https://issues.apache.org/jira/browse/CALCITE-5789">CALCITE-5789</a>]
```

Comment:
It seems to me that these are mainly bug fixes, which IIRC are usually not part of the highlights of the RN (unless it is something major). As far as I'm concerned, I would consider CALCITE-5789 and CALCITE-5401 not that relevant to be considered as a highlight.

Reply:
Remove it.